docs/ovs-pipeline.md

```diff
@@ -414,12 +412,12 @@ to the next tier in precedence, or in the case of the Application tier (lowest
 precedence), go to the [EgressRuleTable].
 
 If the `conjunction` action is matched, packets are "allowed" or "dropped"
-based on the `action` field of the CNP rule. If allowed, they follow a similar
+based on the `action` field of the policy rule. If allowed, they follow a similar
 path as described in the following [EgressRuleTable] section.
 
-Unlike the default of K8s NetworkPolicies, ClusterNetworkPolicy has no such
-default rules. Hence, they are evaluated as-is, and there is no need for a
-CnpEgressDefaultTable.
+Unlike the default of K8s NetworkPolicies, ClusterNetworkPolicy or Antrea NetworkPolicy
+has no such default rules. Hence, they are evaluated as-is, and there is no need for a
```

Comment:
ClusterNetworkPolicy and Antrea NetworkPolicy have no such default rules

Reply:
Done.